bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -459,7 +459,12 @@ <h2 class="tween">
         <div class="intro tween">
           {% if l10n_has_tag('mwc_2015') %}
             <h4>{{ _('A superior mobile experience') }}</h4>
-            <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+            <div class="play-button"> 
+              <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+              <small class="download-other">
+                <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+              </small>
```

Comment:
Good catch, that will make things confusing. Also, after some additional testing, turns out this blue is just a terrible background color ;) The contrast here is definitely way to low and the legibility is not good at all but, let's leave that as is for now, and I will open a bug to explore how we can resolve this.